draft-ietf-jsonpath-base.md

```diff
@@ -958,19 +961,29 @@ In the resultant nodelist:
 
 Children of an object may occur in any order, since JSON objects are unordered.
 
-### Union Selector
+### List Selector
+
+The list selector allows combining member names, array indices, and
+slices in a single selector.
+
+Note: The list selector was called "union selector" in
+{{JSONPath-orig}}, as it was intended to solve use cases addressed by
+the union selector in XPath.
+However, the term "union" implies the set operation that involves
+merging input sets while avoiding duplicates, so the concept was
+renamed into "list selector".
 
 #### Syntax
 {: unnumbered}
 
-The union selector is syntactically related to the
-`index-selector`.
+The list selector is syntactically related to the
+`index-selector` and the `slice-selector`.
```

Comment:
What's the relationship between `index-selector` and member names? That might answer my question from above as well.  Member names are again mentioned on 986.  I just find it confusing to have two terms in some descriptions and three in others.

Maybe this could be addressed in a follow-up PR?

Reply:
At this point, the reader should already know what an index selector is.